src/filesystem/Parser.cpp

```diff
@@ -107,6 +107,8 @@ static struct options {
   bool allowOther;
 
   int qsfsMultiThread;       // internal use only to turn on multi thread
+
+  mode_t fallbackMode;     // default fallback mode
```

Comment:
What does fallback means here? Seems "fallbackMode" is not a great name. what we are doing here is to provide a option to specifying a file mode (OCTAL-MODE format), I think we could just name it fileMode or some other name which can clearly show the intention.